qiskit/pulse/macros.py

```diff
@@ -52,10 +54,23 @@ def measure(
         PulseError: If both ``inst_map`` or ``meas_map``, and ``backend`` is None.
     """
     schedule = Schedule(name=f"Default measurement schedule for qubits {qubits}")
+
     try:
-        inst_map = inst_map or backend.defaults().instruction_schedule_map
-        meas_map = meas_map or backend.configuration().meas_map
+        # backend is V2.
+        if backend is None or hasattr(backend, "target"):
+            if target is None:
+                target = backend.target
```

Comment:
I think the logic here is problematic.
If you provide as arguments `inst_map` and `meas_map`, the macro has all the information it needs, but the condition here will raise the exception.